portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase

```diff
@@ -418,4 +534,184 @@ definition {
+		}
+
+		task ("Then the New Item Action modal is displayed") {
+			AssertElementPresent(
+				key_title = "New Item Action",
+				locator1 = "Header#H2_TITLE");
+		}
+
+		task ("Then the Icon field is not mandatory") {
+			AssertElementNotPresent(
+				key_fieldName = "Icon",
+				locator1 = "DataSet#MANDATORY_FIELD");
+		}
+
+		task ("When the user clicks on the Icon field") {
+			Click(
+				key_fieldLabel = "Icon",
+				locator1 = "Select#GENERIC_SELECT_FIELD");
+		}
+
+		task ("Then a dropdown list all the icons in clay with the name and the corresponding icon") {
+			Select(
+				key_fieldLabel = "Icon",
+				locator1 = "Select#GENERIC_SELECT_FIELD",
+				value1 = "accessibility");
+
+			AssertElementPresent(
+				key_text = "pt-br",
+				locator1 = "Icon#ANY");
+		}
+	}
+
+	@description = "LPS-192149. Verify if the Confirmation Message field is not mandatory and is localized"
+	@priority = 4
+	test LocalizedConfirmationMessageField {
+		task ("And clicks on Create Item Action button") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("And selects the Link option") {
+			Select(
+				key_fieldLabel = "Type",
+				locator1 = "Select#GENERIC_SELECT_FIELD",
+				value1 = "Link");
+		}
+
+		task ("Then the Confirmation Message field is present") {
+			AssertElementPresent(
+				key_placeholder = "Add a message here.",
+				locator1 = "FrontendDataSet#PLACEHOLDER");
+		}
+	}
+
+	@description = "LPS-192149. Verify if the Label field is mandatory and localized in Action"
+	@priority = 5
+	test MandatoryAndLocalizedLabelField {
+		task ("And clicks on Create Item Action button") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("Then the New Item Action modal is displayed ") {
+			AssertElementPresent(
+				key_title = "New Item Action",
+				locator1 = "Header#H2_TITLE");
+		}
+
+		task ("And the Label field is mandatory") {
+			AssertElementPresent(
+				key_fieldName = "Label",
+				locator1 = "DataSet#MANDATORY_FIELD");
+		}
+
+		task ("And Label field is localized ") {
+			AssertElementPresent(
+				key_field = "Label",
+				key_localizedLanguage = "en_US",
+				locator1 = "DataSet#LOCALIZED_FIELD");
+		}
+	}
+
+	@description = "LPS-192149. Verify if the Type field is mandatory and shows the Link option"
+	@priority = 5
+	test MandatoryTypeField {
+		task ("And clicks on plus button") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("Then the New Item Action modal is displayed") {
+			AssertElementPresent(
+				key_title = "New Item Action",
+				locator1 = "Header#H2_TITLE");
+		}
+
+		task ("Then the Type field is mandatory") {
+			AssertElementPresent(
+				key_fieldName = "Type",
+				locator1 = "DataSet#MANDATORY_FIELD");
+		}
+
+		task ("When the user clicks on the Type field") {
+			Click(
+				key_fieldLabel = "Type",
+				locator1 = "Select#GENERIC_SELECT_FIELD");
+		}
+
+		task ("Then a dropdown is displayed containing the options: Link") {
+			AssertElementPresent(
+				key_typeName = "Link",
+				locator1 = "Select#TYPE_SELECTED");
+		}
+	}
```

Comment:
The Type filed cannot be empty by definition, so this test is not needed. We can just remove it